azure-devops/azext_devops/dev/pipelines/runs_artifacts.py

```diff
@@ -34,7 +35,7 @@ def run_artifact_list(run_id, organization=None, project=None, detect=None):
     """
     organization, project = resolve_instance_and_project(detect=detect, organization=organization, project=project)
     client = get_build_client(organization)
-    artifacts = client.get_artifacts(run_id, project)
+    artifacts = client.get_artifacts(project=project, build_id=run_id)
```

Comment:
I think it will be good if use named params while calling functions everywhere in our code..
Thoughts? 
@ishitam8  @atbagga

Reply:
Yes, more importantly when referencing external module functions. 
If we want to always do it.. lets check if pylint has such a rule.